index.html

```diff
@@ -0,0 +1,273 @@
+                    Hello I’m a software developer! I can help 
+                    you build a product , feature or 
+                    website Look through some of my work 
+                    and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <div class="connect">
+                    <div id="title">LET’S CONNECT</div>
+                    <ul class="social d-flex justify-start no-pad">
+                    <li>
+                            <a href="https://twitter.com/abiola_arobieke">
+                                <img src="./images/twitter-icon.png" alt="twitter-icon">
+                            <a/> 
+                    </li>
+                    <li>
+                            <a href="https://linkedin.com/in/abiola-arobieke">
+                                <img src="./images/linkedin-icon.png" alt="linkedin-icon">
+                            <a/> 
+                        </li>
+                    <li><a href="#"><img src="./images/medium-icon.png" alt="medium-icon"><a/> </li>
+                    <li> 
+                            <a href="https://github.com/abiola-arobieke">
+                                <img src="./images/github-icon.png" alt="github-icon">
+                            <a/> 
+                        </li>
+                    <li><a href="#"><img src="./images/hand-icon.png" alt="hand-icon"><a/> </li>
+                    </ul>
+                </div>
+                <button class="btn-action" type="button">Get my resume</button> 
+            </div>
+            <div class="skill-set">
+                <div class="tech lang d-flex justify-between">
+                    <h4>Frameworks</h4>
+                    <button type="button"> v </button>
+                </div>    
+                <ul class="d-flex flex-column no-pad">
+                    <li class="d-flex">
+                        <img src="./images/js-logo.png" alt="js-logo">
+                        <span>javascript</span>
+                    </li>
+                    <li class="d-flex">
+                        <img src="./images/html-logo.png" alt="html-logo"> 
+                        <span>HTML</span>
+                    </li>
+                    <li class="d-flex">
+                        <img src="./images/css-logo.png" alt="css-logo">
+                        <span>CSS</span>
+                    </li>
+                </ul>
+                <div>
+
+                    <div>
+                        <div class="tech d-flex justify-between">
+                            <h4>Frameworks</h4>
+                            <button type="button"> > </button>
+                        </div>
+                    </div>
+
+                    <div>
+                        <div class="tech d-flex justify-between">
+                            <h4>Skills</h4>
+                            <button type="button"> > </button>
```

Comment:
- [ ] Based on the template given kindly make adjustments as shown in the picture comparison below. this is suggested as it will be the same when dealing with client projects :

## Figma:
![image](https://user-images.githubusercontent.com/50990925/220268293-bacaac81-8d8d-468b-afb2-802e1d34f9b5.png)

## Your view:
![image](https://user-images.githubusercontent.com/50990925/220268360-edc79c68-2e28-42cd-9778-1b552b477b04.png)